web-registry/src/hooks/useQueryProjectsByIssuer.ts

```diff
@@ -41,7 +41,7 @@ export default function useQueryProjectsByIssuer(issuer: string) {
           setProjects(_projects);
         }
       } catch (error) {
-        console.error(error);
+        console.error(error); // eslint-disable-line no-console
```

Comment:
I hadn't realized it, but a recent pr (https://github.com/regen-network/regen-web/pull/1013/files) overrode the default eslint rules, such as `no-console` and requiring a function return type. There are several `eslint-disable`s to get around that